### PR TITLE
LEAF-3630: Simplify instructions

### PR DESCRIPTION
**Pull request by mgaoVA**
This aims to simplify installation for developers.

1. Are the lines removed necessary? It seems to work without them.
2. Is deployment impacted if the Config.php files are stored in the repository? It would be easier if people didn't need to manually create this.

### CODE REVIEW

**Review by maxrichardjr — APPROVED** [review]

**Comment by shaneodd** [comment]
> This aims to simplify installation for developers.
> 
> 1. Are the lines removed necessary? It seems to work without them.
> 2. Is deployment impacted if the Config.php files are stored in the repository? It would be easier if people didn't need to manually create this.

I would suggest we leave the notes for generating configs for each side, globals I believe are now part of the repo. For the configs I think that will be the plan eventually, Jamie is working on getting everything that would be in the configs into the db since there was a few more things to adjust. I think he can maybe explain that part a bit better than I.

File: docs/InstallationConfiguration.md [hunk 1]
```diff
@@ -77,47 +51,10 @@ class Config
...
-if (!defined('LEAF_NEXUS_URL')) define('LEAF_NEXUS_URL', getenv('APP_URL_NEXUS'));
-if (!defined('HTTP_HOST')) define('HTTP_HOST', getenv('APP_HTTP_HOST'));
-if (!defined('AUTH_URL')) define('AUTH_URL', getenv('APP_URL_AUTH'));
-if (!defined('AUTH_TYPE')) define('AUTH_TYPE', getenv('APP_AUTH_TYPE'));
-if (!defined('CIPHER_KEY')) define('CIPHER_KEY', getenv('APP_CIPHER_KEY'));
-if (!defined('LIB_PATH')) define('LIB_PATH', getenv('APP_LIB_PATH'));
-
-```
-
-Create the following files `sources/DbConfig.php` and `sources/Config.php` and place the following code into them
```
Comment (shaneodd) [inline docs/InstallationConfiguration.md:103]: I would keep this block for now until we move over to using just the db for configs.

**Review by shaneodd — CHANGES_REQUESTED** [review]

**Review by aerinkayne — COMMENTED** [review]
There are currently two files portal side.  If setup will change again in the near future though, I agree it might be better to wait to update.

File: docs/InstallationConfiguration.md (hunk 1 above)
Comment (jampaul3) [reply docs/InstallationConfiguration.md:103]: I'm in agreement with Shane. Changes to the Config files in both the LEAF_Nexus and LEAF_Request_Portal are coming in the next sprint I believe. When those changes come all the data will be pulled from the db and these classes will be updated like all other classes and won't need to be protected by gitignore. The DB_Config is going to go away completely and will no longer be used.

**Comment by jampaul3** [comment]
I believe the changes proposed here are going to change again in the next sprint so I feel we ought to wait until at least then to make these changes. I can make them in my branch that deals with these.

File: docs/InstallationConfiguration.md (hunk 1 above)
Comment (shaneodd) [reply docs/InstallationConfiguration.md:103]: This change will get superseded by this change https://github.com/department-of-veterans-affairs/LEAF/pull/1939

**Review by raphaelfretter — APPROVED** [review]

**Pull request closed without merging**